### PR TITLE
feat(home): add feature console

### PR DESCRIPTION
**Pull request by krisnafirdaus**
# Background 

- add console for user can see number

# Proof of Work

Screenshot 2023-12-14 at 11 16 06

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,6 +84,10 @@ <h1 id="calculator-saving">0</h1>
         }
 
         fetchApi()
+
+        function checkConsole() {
```
Comment (krisnafirdaus) [inline index.html:88]: kak kurang baagua

**Review by indrapurnayasa — APPROVED** [review]

**Review by adhityahardy — APPROVED** [review]
mantap gan!

**Review by ArjunaAcchaDipa — APPROVED** [review]

**Review by naufaalaf — APPROVED** [review]
sip

**Review by Asabito — APPROVED** [review]
udah keren mas, lanjutkan

**Review by FarhanLhant — COMMENTED** [review]
LGTM :)

**Review by faizalheryanto — COMMENTED** [review]
keren banget

**Review by kurniadrajat — COMMENTED** [review]
ok

**Pull request merged**